docs/csharp/quick-starts/hello-world.yml

```diff
@@ -33,8 +33,9 @@ items:
   durationInMinutes: 3
   content: |
     Your first program printed the `string` "Hello World!" on
-    the screen. The rest of this lesson explores working with
-    the `string` type, which represents text in C#.
+    the screen. It used the `Console.WriteLine` method to print that message. `Console` is a type that represents the text console. `WriteLine` is a method that prints a line of text to that text console.
```

Comment:
I think that Console.WriteLine would be best here as a link (\<xref:System.Console.WriteLine?displayProperty=nameWithType>), so that the user can go to it for more information.
text console --> console window
perhaps "WriteLine is a method" --> "WriteLine is a method of the Console type"

Reply:
Thanks for the suggestions @rpetrusha  I just pushed an update with those changes.